composer.json

```diff
@@ -26,17 +26,17 @@
     "silverstripe/assets": "^1@dev",
     "swiftmailer/swiftmailer": "~5.4",
     "symfony/cache": "^3.3@dev",
-    "symfony/config": "^2.8",
+    "symfony/config": "^3.2",
     "symfony/translation": "^2.8",
-    "symfony/yaml": "~2.7",
+    "symfony/yaml": "~3.2",
     "vlucas/phpdotenv": "^2.4"
   },
   "require-dev": {
     "phpunit/phpunit": "^5.7",
     "silverstripe/versioned": "^1.0@dev",
-    "silverstripe/behat-extension": "^2.1.0",
+    "silverstripe/behat-extension": "^3",
     "silverstripe/serve": "dev-master",
-    "silverstripe/testsession": "^2.0.0-alpha3",
+    "silverstripe/testsession": "^3",
```

Comment:
Do we need to explicitly declare this dependency or can we just let behat-extension pull it in? Less is more :-)